backends/src/test_utils/gateway.rs

```diff
@@ -13,7 +13,7 @@ use wiremock::{
 };
 
 use crate::client::{
-    permit::{Error, Organization},
+    permit::{Organization, Result},
```

Comment:
Having `Result` come from permit is weird

Reply:
skill issue :D